packages/js/api-core-tests/data/shared/customer.js

```diff
@@ -29,14 +29,14 @@ const customerBilling = {
 const customerShipping = {
 	first_name: 'Tim',
 	last_name: 'Clark',
-	company: 'Automattic',
+	company: 'Murphy LLC',
```

Comment:
Curious on changing the values here and for the phone number, is this something we could have done in the setup of the test?

Reply:
Yep, just realized that's the better approach. I'll do that on the next change. Thanks!

Reply:
No problem! Thanks for making this change!